OJP/OJP_Trips.xsd

```diff
@@ -674,12 +674,12 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:choice>
-				<xs:element name="TransferMode" type="TransferModesEnumeration">
+				<xs:element name="TransferType" type="TransferTypeEnumeration">
 					<xs:annotation>
-						<xs:documentation>Mode that is used for this interchange between public services.</xs:documentation>
+						<xs:documentation>TYPE that is used for this interchange between public services (TYPE OF TRANSFER).</xs:documentation>
 					</xs:annotation>
 				</xs:element>
-				<xs:element name="ContinuousMode" type="ContinuousModesEnumeration">
+				<xs:element name="AlternativeModeOfOperation" type="AlternativeModesOfOperationEnumeration">
```

Comment:
I don't see how parkAndRideSharing, longDistancePooling, commuterPooling, parkAndRidePooling, hotelShuttle, companyShuttle, school work for a tranferLeg.

Reply:
OK. will work on the lists again.